pkg/apis/jaegertracing/v1/jaeger_types.go

```diff
@@ -261,6 +261,10 @@ type JaegerIngressOpenShiftSpec struct {
 
 	// +optional
 	HtpasswdFile string `json:"htpasswdFile,omitempty"`
+
+	// SkipLogout tells the operator to not automatically add a "Log Out" menu option to the custom Jaeger configuration
+	// +optional
+	SkipLogout *bool `json:"skipLogout,omitempty"`
```

Comment:
does it need to be a pointer? We probably want the default to be false.

Reply:
For all new properties, we probably want a tri-state value: `nil` (not set), `true`, `false`. This way, the property doesn't get persisted as part of the CR, and we can have different defaults depending on the platform.